stimela/tests/acceptance_tests/stimela-test-ngc417.py

```diff
@@ -563,9 +563,9 @@ def testEndToEndReduction(self):
                         "msname"        :   MS,
                         "channelrange"  :   chans,               #Other channels don't have any data   
                         "weight"        :   "briggs 0",               # Use Briggs weighting to weigh visibilities for imaging
-                        "npix"          :   1026,                   # Image size in pixels
-                        "trim"          :   1024,                    # To avoid aliasing
-                        "cellsize"      :   2.5,                      # Size of each square pixel
+                        "npix"          :   8196,                   # Image size in pixels
```

Comment:
These images are too big for a test!

Reply:
Lets first get it back to passing before worrying about image size